openshift_tools/ircbot/sopel.cfg.example

```diff
@@ -9,6 +9,10 @@ admins = admin1, admin2
 channels = #mybot
 exclude = url, ipython
 extra = /absolute/path/to/openshift-tools/openshift_tools/trello, /absolute/path/to/openshift-tools/openshift_tools/openshift_sre
+auth_method = userserv
```

Comment:
@aweiteka Thoughts on changing our requirements.txt to use my fork that allows this to work? https://github.com/wgordon17/sopel/tree/add-userserv-auth